lib/scsi_defs.h

```diff
@@ -389,11 +389,11 @@ static_assert(sizeof(Read6Command) == 5);
 // SCSI Reference Manual Table 97
 // https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
 struct Read10Command {
-  uint8_t rd_protect : 3;  // read protect bit
-  bool dpo : 1;            // disable page output bit
-  bool fua : 1;            // Forced Unit access bit
-  bool rarc : 1;           // Rebuild Assist Recovery bit
-  bool fua_nv : 1;         // Forced Access Unit Non-Volatile
+  uint8_t rdprotect : 3;  // read protect bit
```

Comment:
renaming because the field is RDPROTECT so the underscore didn't make sense in the first place.